option.go

```diff
@@ -418,9 +420,10 @@ func GrpcRunnerWithOptions(name, target string, opts ...grpcRunnerOption) Option
 		}
 		delete(bk.runnerErrs, name)
 		r := &grpcRunner{
-			name:   name,
-			target: target,
-			mds:    map[string]protoreflect.MethodDescriptor{},
+			name:            name,
+			target:          target,
+			mds:             map[string]protoreflect.MethodDescriptor{},
+			traceHeaderName: strings.ToLower(defaultTraceHeaderName),
```

Comment:
Same as https://github.com/k1LoW/runn/pull/742#discussion_r1441145862

```suggestion
			traceHeaderName: defaultTraceHeaderName,
```